Userland/Libraries/LibC/CMakeLists.txt

```diff
@@ -111,6 +111,7 @@ elseif ("${SERENITY_ARCH}" STREQUAL "x86_64")
 endif()
 
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-unknown-warning-option")
+add_compile_options(-ftls-model=initial-exec)
```

Comment:
What about the other system libraries that are components of/required by LibC? Should we add initial exec to them too?

Reply:
As far as I can tell, they do not reference any thread-local symbols at the moment. To be on the safe side, I moved the option to the `serenity_libc` function.